### PR TITLE
Basic GIT Intro, features of VS CODE(#39512)

### PR DESCRIPTION
**Pull request by Ankush-Negi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Develop branch created
+# Feature Branch Created
```
Comment (VisaaliSuccessive) [inline README.md]: remove these lines from readme

File: VS-CODE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Features of VS Code:-
```
Comment (VisaaliSuccessive) [inline VS-CODE.md]: make this as heading 1

File: VS-CODE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+   1. Emmet Abbreviations(shortcuts come built-in to VS Code)
+   2. The Integrated CLI (Command Line Interface)- To save switching between windows
+   3. ESLint-helps you spot errors in your code and fix them as you write
+   4. Prettier-Prettier is an opinionated code-formatter. It prescribes
```
Comment (VisaaliSuccessive) [inline VS-CODE.md]: add numbered list without tabs using markdown syntax

File: VS-CODE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            a certain formatting style as the correct one, but its popularity is making 
+            its rules something of an established standard 
+            for JavaScript, CSS, and increasing numbers of other languages.
+   5. Text Wrap- Used to fit the page inside the screen so as the user don't have to
+            use the scroll bar to se the entire script.
+     
+Useful plugins for VS Code:-
```
Comment (VisaaliSuccessive) [inline VS-CODE.md]: make this as heading 2

File: VS-CODE.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    1. Prettier- Prettier doesn’t need much of an introduction.
```
Comment (VisaaliSuccessive) [inline VS-CODE.md]: make each extension's name bold

File: VS-CODE.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        from VS Code, while it also supports live reloading.
+    5. REST Client- REST client might be one of the best extensions I have used in a while, as it
+        allows you to send HTTP requests and view the responses directly in VS Code.
+    6. Babel JavaScript- It supports ES201x, React and JSX syntax highlighting, which I guess is 
+        something most people working with any kind of modern web application would like to have. 
+        It might not look like much initially, but it does make a difference in the long run.
+    7. Fira Code- Fira Code is one of the best programming fonts with ligatures that can help you
+        scan code a lot faster once you get used to it.
+    8. Quokka.js- Quokka.js is a playground for JavaScript and TypeScript. It runs your code 
+        immediately as you type and displays various execution results in your code editor.
```
Comment (VisaaliSuccessive) [inline VS-CODE.md]: make all points using markdown syntax

File: README.md (hunk 1 above)
Comment (Ankush-Negi) [reply README.md]: Done

File: VS-CODE.md (hunk 2 above)
Comment (Ankush-Negi) [reply VS-CODE.md]: Done

File: VS-CODE.md (hunk 3 above)
Comment (Ankush-Negi) [reply VS-CODE.md]: Done

File: VS-CODE.md (hunk 4 above)
Comment (Ankush-Negi) [reply VS-CODE.md]: Done

File: VS-CODE.md (hunk 5 above)
Comment (Ankush-Negi) [reply VS-CODE.md]: Done

File: VS-CODE.md (hunk 6 above)
Comment (Ankush-Negi) [reply VS-CODE.md]: Done

**Review by VisaaliSuccessive — APPROVED** [review]

**Pull request merged**